manifests/params.pp

```diff
@@ -13,6 +13,7 @@
   # by cvmfs. If you are managing autofs elsewhere set to false.
   # This being the string 'true' and not boolean true is a hiera bug.
   $config_automaster      = hiera('cvmfs_config_automaster','true')
+  $manage_autofs_service  = hiera('cvmfs_manage_autofs_service','true')
```

Comment:
Hmmm this is backwards incompatible if someone has set config_automaster false.

In params.pp $manage_autofs_service should default to $cvmfs::config_automaster I think? Not sure if that works.